scripts/bert/bertpass.cc

```diff
@@ -460,14 +496,23 @@ MXReturnValue custom_pass(mxnet::ext::Graph *g,
       }
 #endif
       // concatenate bias terms
-      for (int e =0; e < shape0; ++e) {
-        qkv_bias_data[e] = query_bias_data[e];
+      int counter = 0;
+      for (int e = 0; e < num_heads * 3; e += 3) {
+          for (int h = e * head_dimension; h < e * head_dimension + head_dimension; h++) {
+            qkv_bias_data[h] = query_bias_data[counter++];
+          }
       }
-      for (int e=0; e < shape0; ++e) {
-        qkv_bias_data[shape0 + e] = key_bias_data[e];
+      counter = 0;
+      for (int e = 1; e < num_heads * 3; e += 3) {
+          for (int h = e * head_dimension; h < e * head_dimension + head_dimension; h++) {
+            qkv_bias_data[h] = key_bias_data[counter++];
+          }
       }
-      for (int e=0; e < shape0; ++e) {
-        qkv_bias_data[2 * shape0 + e] = value_bias_data[e];
+      counter = 0;
+      for (int e = 2; e < num_heads * 3; e += 3) {
```

Comment:
Could you please tell me whether we can avoid proliferation of those loops? It looks like that these loops are "identical", only a few parameters are different.

Reply:
Done, please check